### PR TITLE
filters/scheduler: add error tag to span on lifo error

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: filters/scheduler/lifo_test.go [hunk 1]
```diff
@@ -290,3 +300,88 @@ func TestNewLIFO(t *testing.T) {
...
+	assert.Equal(t, map[uint16]int{
+		// 20 request in total, of which:
+		200: 5, // went straight to the backend
+		502: 7, // were queued and timed out as backend latency is greater than scheduling timeout
+		503: 8, // were refused due to full queue
+	}, codes)
+
+	metrics.WithCounters(func(counters map[string]int64) {
+		assert.Equal(t, int64(7), counters["lifo.aroute.error.timeout"])
+		assert.Equal(t, int64(8), counters["lifo.aroute.error.full"])
```
Comment (AlexanderYastrebov) [inline filters/scheduler/lifo_test.go:371]: Were introduced by #1827 without test

File: filters/scheduler/lifo_test.go [hunk 2]
```diff
@@ -290,3 +300,88 @@ func TestNewLIFO(t *testing.T) {
...
 	}
 }
+
+func TestLifoErrors(t *testing.T) {
+	backend := httptest.NewServer(http.HandlerFunc(func(_ http.ResponseWriter, _ *http.Request) {
+		time.Sleep(time.Second)
+	}))
+	defer backend.Close()
+
+	doc := fmt.Sprintf(`aroute: * -> lifo(5, 7, "100ms") -> "%s"`, backend.URL)
```
Comment (szuecs) [inline filters/scheduler/lifo_test.go:310]: let's see if the timeout 100ms is not too short and result in flaky test

**Comment by szuecs** [comment]
:+1:

File: filters/scheduler/lifo_test.go (hunk 2 above)
Comment (AlexanderYastrebov) [reply filters/scheduler/lifo_test.go:310]: I hope it should be fine - backend latency is 10x == 1s

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**